source/extensions/filters/listener/proxy_protocol/proxy_protocol_header.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+namespace Envoy {
+namespace Extensions {
+namespace ListenerFilters {
+namespace ProxyProtocol {
+
+//
```

Comment:
nit: No leading "//" on block comments. In general, please try to wrap comments around 100 cols (this comment applies elsewhere).